crates/service_utils/src/db/pgschema_manager.rs

```diff
@@ -58,7 +58,9 @@ impl From<Vec<ConnectionConfig>> for PgSchemaManager {
                 Pool::builder()
                     .max_size(config.count)
                     .build(manager)
-                    .expect(format!("Invalid config provided, {}", config.name).as_str()),
+                    .unwrap_or_else(|_| {
```

Comment:
expect does the same thing right?

Reply:
```text
warning: use of `expect` followed by a function call
  --> crates/service_utils/src/db/pgschema_manager.rs:61:22
   |
61 |                     .expect(format!("Invalid config provided, {}", config.name).as_str()),
   |                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try: `unwrap_or_else(|_| panic!("Invalid config provided, {}", config.name))`
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#expect_fun_call
   = note: `#[warn(clippy::expect_fun_call)]` on by default
```

`expect` does the same thing, yes. It's only that the `format!()` will always be called (and thus, an allocation always happens) with `expect()`, while `unwrap_or_else()` involves lazy evaluation, so the `format!()` will only be called in case of errors.